src/Network/Network/ApplicationGateway/RewriteRuleCondition/NewAzureApplicationGatewayRewriteRuleConditionCommand.cs

```diff
@@ -0,0 +1,29 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.Network.Models;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Network
+{
+    [Cmdlet("New", ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "ApplicationGatewayRewriteRuleCondition"), OutputType(typeof(PSApplicationGatewayRewriteRuleCondition))]
```

Comment:
@sobaner please make sure there is a corresponding markdown help file for this cmdlet -- you can find how to generate this file by following [this document](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)